src/dataflow/src/render/reduce.rs

```diff
@@ -1213,6 +1213,20 @@ enum AccumInner {
         /// The number of non-NULL values observed.
         non_nulls: isize,
     },
+    /// Accumulates float values.
+    Float {
```

Comment:
We need to watch out that the size of `AccumInner` doesn't get too big, since it might be used in a lot of places.

Reply:
Not sure how big is too big. I think currently the Numeric variant is 95 bytes wide (or more if there is padding) and the Float one 48